keyboards/lily58/keymaps/gshmu/rules.mk

```diff
@@ -0,0 +1,6 @@
+VIA_ENABLE = yes
+OLED_ENABLE = no
```

Comment:
if you're not using the oled feature, you may want to just remove the oled code from your keymap, completely.